dm/ui/locales/en.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
Using YAML may be more convenient, it supports comments and multiple lines, and easy multiple layers.

Reply:
A vite plugin is used to automatically scan all files and extract keys to json files during dev, I'll add yaml support later.